docs/design_docs/010-market-params.md

```diff
@@ -0,0 +1,163 @@
+# ADR 010: Market Parameters
+
+## Changelog
+
+- Jul 4, 2022: Max collateral utilization added to ADR-003 (@robert-zaremba)
+- Jul 18, 2022: Initial draft moved discussion to ADR-101 (@toteki)
```

Comment:
```suggestion
- Jul 18, 2022: Initial draft moved discussion to ADR-010 (@toteki)
```

Reply:
Note: Fixed in document moving/renaming PR